dict.de.css

```diff
@@ -89,3 +89,11 @@ a.action.load-more::before {
 .filter-wrapper .filter:first-child::before {
   content: "Filter";
 }
+
+.filter .products button::before {
+  content: 'Products';
```

Comment:
Please use double quotes for all CSS strings to stay consistent with the rest of the file.

Reply:
changed, oversee was working in another project today using single Thanks!